README.md

```diff
@@ -1,7 +1,12 @@
 # Google Software Product Sprint
 
+<<<<<<< HEAD
+<<<<<<< HEAD
+This repo contains the projects you'll work on throughout SPS.
+=======
 
 This repo contains Renaud Noubieptie's portfolio and SPS projects.
+>>>>>>> 6569359faef796bf6816944e082d36eab62d92c8
```

Comment:
This line marks the end of the merge conflict, the long string is the id of the branch you are having conflict with.

Reply:
Hi Chen, I believe I now resolved the merge conflict. Thank you for the feedback please keep it coming be as pick and rigorous as possible, thank you for your help.